conf.py

```diff
@@ -1360,7 +1360,7 @@
 USE_BUNDLES = False
 
 # Plugins you don't want to use. Be careful :-)
-# DISABLED_PLUGINS = ["render_galleries"]
+DISABLED_PLUGINS = ["robots"]
```

Comment:
Wait, you want to disable Rbots? That sounds like the opposite of what we want, have I misunderstood something here?

Reply:
Urgh, I think it was my misunderstanding. This isn't particularly well documented but I was getting a warning that seemed like it was related to the exclusions. Disabling robots seemed to fix that while still generating `output/robots.txt` and the sitemap as expected.

On second look that warning seems to have gone away and was in fact related to the value of `SITE_URL`. Not sure if the fact nikola was still generating robots.txt with this is a bug or not.